javascript/index.js

```diff
@@ -40,23 +44,41 @@ function clearSplits() {
 
 function setStopBtn() {
   // ... your code goes here
+  btnLeft.innerText = "STOP"
+  btnLeft.className = "btn stop"
 }
 
 function setSplitBtn() {
   // ... your code goes here
+  btnRight.innerText = "SPLIT"
+  btnRight.className = "btn split"
 }
 
 function setStartBtn() {
   // ... your code goes here
+  btnLeft.innerText = "START"
+  btnLeft.className = "btn start"
 }
 
 function setResetBtn() {
   // ... your code goes here
+  btnRight.innerText = "RESET"
+  btnRight.className = "btn reset"
 }
 
 // Start/Stop Button
 btnLeft.addEventListener('click', () => {
   // ... your code goes here
+  if (btnLeft.innerText==="START") {
+    setStopBtn()
+    setSplitBtn()
+    //startClick() here??
```

Comment:
yes! this was the main part to conecting your two files, you want to invoke the method startClick() that starts your chronometer here. the way to do it is to access the chronometer object and run it's method. so:
`chronometer.startClick(printTime)`

Reply:
and why are we sending printTime as a parameter? this printTime function will be the main function that updates the DOM so it will change the numbers you see in the chronometer. So you want this to run not just one time, but every second. To do this you send the function (via parameter) as a callback to the startClick method and then inside the interval is when you want to execute the function.